users/test/test_views.py

```diff
@@ -7,12 +7,108 @@
 from rest_framework.reverse import reverse
 from rest_framework.test import APIRequestFactory, APITestCase
 
-from users.views import PasswordResetAPIView, PasswordResetEmailView
+from users.views import (
+    PasswordResetAPIView,
+    PasswordResetEmailView,
+    UserDetail,
+    UserList,
+)
 
 fake = Faker()
 User = get_user_model()
+
 password_reset_email_view = PasswordResetEmailView.as_view()
 password_reset_api_view = PasswordResetAPIView.as_view()
+user_detail = UserDetail.as_view()
+user_list = UserList.as_view()
```

Comment:
Use Request Factory or client to make API calls instead of calling the view directly